From 3c708a0fa52cc3179a8599114a464892d04f6b09 Mon Sep 17 00:00:00 2001
From: Abdoulaye Walsimou Gaye <awg@embtoolkit.org>
Date: Sat, 18 Oct 2014 20:47:34 +0200
Subject: [PATCH] [Embtk] Fix build on BSD systems and clang

Signed-off-by: Abdoulaye Walsimou Gaye <awg@embtoolkit.org>
---
 include/safe-ctype.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/safe-ctype.h b/include/safe-ctype.h
index 8665349..4de5b23 100644
--- a/include/safe-ctype.h
+++ b/include/safe-ctype.h
@@ -119,7 +119,8 @@ extern const unsigned char  _sch_tolower[256];
    including another system header (for instance gnulib's stdint.h).
    So we include ctype.h here and then immediately redefine its macros.  */
 
-#if !defined(__FreeBSD__) || !defined(__clang__)
+#if defined(__linux__) && !defined(__clang__)
+
 #include <ctype.h>
 #undef isalpha
 #define isalpha(c) do_not_use_isalpha_with_safe_ctype
-- 
2.0.3

