diff -Nuar binutils-2.19.1/bfd/Makefile.am binutils-2.19.1-patched/bfd/Makefile.am
--- binutils-2.19.1/bfd/Makefile.am	2009-02-02 13:44:34.000000000 +0300
+++ binutils-2.19.1-patched/bfd/Makefile.am	2015-08-08 20:31:09.178854470 +0300
@@ -10,7 +10,7 @@
 CSEARCH = -I. -I$(srcdir) -I$(INCDIR)
 MKDEP = gcc -MM
 
-SUBDIRS = doc po
+SUBDIRS = po
 
 bfddocdir = doc
 bfdlibdir = @bfdlibdir@
diff -Nuar binutils-2.19.1/bfd/Makefile.in binutils-2.19.1-patched/bfd/Makefile.in
--- binutils-2.19.1/bfd/Makefile.in	2009-02-02 13:44:39.000000000 +0300
+++ binutils-2.19.1-patched/bfd/Makefile.in	2015-08-08 20:31:09.178854470 +0300
@@ -275,7 +275,7 @@
 INCDIR = $(srcdir)/../include
 CSEARCH = -I. -I$(srcdir) -I$(INCDIR)
 MKDEP = gcc -MM
-SUBDIRS = doc po
+SUBDIRS = po
 bfddocdir = doc
 bfdlib_LTLIBRARIES = libbfd.la
 AM_CFLAGS = $(WARN_CFLAGS)
diff -Nuar binutils-2.19.1/gas/Makefile.am binutils-2.19.1-patched/gas/Makefile.am
--- binutils-2.19.1/gas/Makefile.am	2009-01-14 11:50:17.000000000 +0300
+++ binutils-2.19.1-patched/gas/Makefile.am	2015-08-08 20:31:09.178854470 +0300
@@ -3,7 +3,7 @@
 AUTOMAKE_OPTIONS = 1.8 cygnus dejagnu
 ACLOCAL_AMFLAGS = -I .. -I ../config -I ../bfd
 
-SUBDIRS = doc po
+SUBDIRS = po
 # Automake should figure this out on its own.  It doesn't, because
 # of the "cygnus" option.  But distclean still wants it.
 DIST_SUBDIRS = $(SUBDIRS)
diff -Nuar binutils-2.19.1/gas/Makefile.in binutils-2.19.1-patched/gas/Makefile.in
--- binutils-2.19.1/gas/Makefile.in	2009-01-14 11:50:17.000000000 +0300
+++ binutils-2.19.1-patched/gas/Makefile.in	2015-08-08 20:31:09.178854470 +0300
@@ -262,7 +262,7 @@
 te_file = @te_file@
 AUTOMAKE_OPTIONS = 1.8 cygnus dejagnu
 ACLOCAL_AMFLAGS = -I .. -I ../config -I ../bfd
-SUBDIRS = doc po
+SUBDIRS = po
 # Automake should figure this out on its own.  It doesn't, because
 # of the "cygnus" option.  But distclean still wants it.
 DIST_SUBDIRS = $(SUBDIRS)
diff -Nuar binutils-2.19.1/ld/Makefile.am binutils-2.19.1-patched/ld/Makefile.am
--- binutils-2.19.1/ld/Makefile.am	2008-12-23 16:54:51.000000000 +0300
+++ binutils-2.19.1-patched/ld/Makefile.am	2015-08-08 20:41:58.242073005 +0300
@@ -91,10 +91,10 @@
   fi`
 
 noinst_PROGRAMS = ld-new
-info_TEXINFOS = ld.texinfo
-ld_TEXINFOS = configdoc.texi
-noinst_TEXINFOS = ldint.texinfo
-man_MANS = ld.1
+#info_TEXINFOS = ld.texinfo
+#ld_TEXINFOS = configdoc.texi
+#noinst_TEXINFOS = ldint.texinfo
+#man_MANS = ld.1
 
 AM_MAKEINFOFLAGS = -I $(srcdir) -I $(BFDDIR)/doc -I ../bfd/doc \
 		   -I $(top_srcdir)/../libiberty
@@ -1815,14 +1815,14 @@
 # Build the man page from the texinfo file
 # The sed command removes the no-adjust Nroff command so that
 # the man output looks standard.
-ld.1: $(srcdir)/ld.texinfo configdoc.texi
-	touch $@
-	-$(TEXI2POD) $(MANCONF) < $(srcdir)/ld.texinfo > ld.pod
-	-($(POD2MAN) ld.pod | \
-	        sed -e '/^.if n .na/d' > $@.T$$$$ && \
-	        mv -f $@.T$$$$ $@) || \
-	        (rm -f $@.T$$$$ && exit 1)
-	rm -f ld.pod
+#ld.1: $(srcdir)/ld.texinfo configdoc.texi
+#	touch $@
+#	-$(TEXI2POD) $(MANCONF) < $(srcdir)/ld.texinfo > ld.pod
+#	-($(POD2MAN) ld.pod | \
+#	        sed -e '/^.if n .na/d' > $@.T$$$$ && \
+#	        mv -f $@.T$$$$ $@) || \
+#	        (rm -f $@.T$$$$ && exit 1)
+#	rm -f ld.pod
 
 MAINTAINERCLEANFILES = configdoc.texi
 
@@ -1917,12 +1917,14 @@
 EXTRA_DIST = ldgram.c ldgram.h ldlex.c emultempl/spu_ovl.o_c \
            deffilep.c deffilep.h $(man_MANS)
 diststuff: info $(EXTRA_DIST)
-all: info ld.1
+all:
+#info ld.1
 
 # Both info (ld.info) and ld.1 depend on configdoc.texi.
 # But info isn't a direct target. Make info-recursive to depend on
 # ld.1 to support parallel build.
-info-recursive: ld.1
+info-recursive:
+#ld.1
 
 DISTCLEANFILES = tdirs site.exp site.bak stringify.sed
 distclean-local:
diff -Nuar binutils-2.19.1/ld/Makefile.in binutils-2.19.1-patched/ld/Makefile.in
--- binutils-2.19.1/ld/Makefile.in	2008-12-23 16:54:52.000000000 +0300
+++ binutils-2.19.1-patched/ld/Makefile.in	2015-08-08 20:43:41.006582592 +0300
@@ -105,11 +105,11 @@
 INFO_DEPS = ld.info
 TEXINFO_TEX = $(top_srcdir)/../texinfo/texinfo.tex
 am__TEXINFO_TEX_DIR = $(top_srcdir)/../texinfo
-DVIS = ld.dvi
-PDFS = ld.pdf
-PSS = ld.ps
-HTMLS = ld.html
-TEXINFOS = ld.texinfo
+#DVIS = ld.dvi
+#PDFS = ld.pdf
+#PSS = ld.ps
+#HTMLS = ld.html
+#TEXINFOS = ld.texinfo
 TEXI2PDF = $(TEXI2DVI) --pdf --batch
 MAKEINFOHTML = $(MAKEINFO) --html
 AM_MAKEINFOHTMLFLAGS = $(AM_MAKEINFOFLAGS)
@@ -343,10 +343,10 @@
     fi; \
   fi`
 
-info_TEXINFOS = ld.texinfo
-ld_TEXINFOS = configdoc.texi
-noinst_TEXINFOS = ldint.texinfo
-man_MANS = ld.1
+#info_TEXINFOS = ld.texinfo
+#ld_TEXINFOS = configdoc.texi
+#noinst_TEXINFOS = ldint.texinfo
+#man_MANS = ld.1
 AM_MAKEINFOFLAGS = -I $(srcdir) -I $(BFDDIR)/doc -I ../bfd/doc \
 		   -I $(top_srcdir)/../libiberty
 
@@ -844,8 +844,8 @@
 distclean-libtool:
 	-rm -f libtool
 
-ld.info: ld.texinfo $(ld_TEXINFOS)
-	restore=: && backupdir="$(am__leading_dot)am$$$$" && \
+#ld.info: ld.texinfo $(ld_TEXINFOS)
+#	restore=: && backupdir="$(am__leading_dot)am$$$$" && \
 	rm -rf $$backupdir && mkdir $$backupdir && \
 	if ($(MAKEINFO) --version) >/dev/null 2>&1; then \
 	  for f in $@ $@-[0-9] $@-[0-9][0-9] $(@:.info=).i[0-9] $(@:.info=).i[0-9][0-9]; do \
@@ -862,19 +862,19 @@
 	fi; \
 	rm -rf $$backupdir; exit $$rc
 
-ld.dvi: ld.texinfo $(ld_TEXINFOS) 
-	TEXINPUTS="$(am__TEXINFO_TEX_DIR)$(PATH_SEPARATOR)$$TEXINPUTS" \
+#ld.dvi: ld.texinfo $(ld_TEXINFOS) 
+#	TEXINPUTS="$(am__TEXINFO_TEX_DIR)$(PATH_SEPARATOR)$$TEXINPUTS" \
 	MAKEINFO='$(MAKEINFO) $(AM_MAKEINFOFLAGS) $(MAKEINFOFLAGS) -I $(srcdir)' \
 	$(TEXI2DVI) -o $@ `test -f 'ld.texinfo' || echo '$(srcdir)/'`ld.texinfo
 
-ld.pdf: ld.texinfo $(ld_TEXINFOS) 
-	TEXINPUTS="$(am__TEXINFO_TEX_DIR)$(PATH_SEPARATOR)$$TEXINPUTS" \
+#ld.pdf: ld.texinfo $(ld_TEXINFOS) 
+#	TEXINPUTS="$(am__TEXINFO_TEX_DIR)$(PATH_SEPARATOR)$$TEXINPUTS" \
 	MAKEINFO='$(MAKEINFO) $(AM_MAKEINFOFLAGS) $(MAKEINFOFLAGS) -I $(srcdir)' \
 	$(TEXI2PDF) -o $@ `test -f 'ld.texinfo' || echo '$(srcdir)/'`ld.texinfo
 
-ld.html: ld.texinfo $(ld_TEXINFOS) 
-	rm -rf $(@:.html=.htp)
-	if $(MAKEINFOHTML) $(AM_MAKEINFOHTMLFLAGS) $(MAKEINFOFLAGS) -I $(srcdir) \
+#ld.html: ld.texinfo $(ld_TEXINFOS) 
+#	rm -rf $(@:.html=.htp)
+#	if $(MAKEINFOHTML) $(AM_MAKEINFOHTMLFLAGS) $(MAKEINFOFLAGS) -I $(srcdir) \
 	 -o $(@:.html=.htp) `test -f 'ld.texinfo' || echo '$(srcdir)/'`ld.texinfo; \
 	then \
 	  rm -rf $@; \
@@ -2636,14 +2636,14 @@
 # Build the man page from the texinfo file
 # The sed command removes the no-adjust Nroff command so that
 # the man output looks standard.
-ld.1: $(srcdir)/ld.texinfo configdoc.texi
-	touch $@
-	-$(TEXI2POD) $(MANCONF) < $(srcdir)/ld.texinfo > ld.pod
-	-($(POD2MAN) ld.pod | \
+#ld.1: $(srcdir)/ld.texinfo configdoc.texi
+#	touch $@
+#	-$(TEXI2POD) $(MANCONF) < $(srcdir)/ld.texinfo > ld.pod
+#	-($(POD2MAN) ld.pod | \
 	        sed -e '/^.if n .na/d' > $@.T$$$$ && \
 	        mv -f $@.T$$$$ $@) || \
 	        (rm -f $@.T$$$$ && exit 1)
-	rm -f ld.pod
+#	rm -f ld.pod
 mostlyclean-local:
 	-rm -rf tmpdir
 
@@ -2720,7 +2720,8 @@
 # cygnus option.
 install-data-local: install-info
 diststuff: info $(EXTRA_DIST)
-all: info ld.1
+all:
+#info ld.1
 
 # Both info (ld.info) and ld.1 depend on configdoc.texi.
 # But info isn't a direct target. Make info-recursive to depend on
